lightwood/data/cleaner.py

```diff
@@ -103,7 +103,8 @@ def cleaner(
         data: pd.DataFrame, dtype_dict: Dict[str, str],
         pct_invalid: float, ignore_features: List[str],
         identifiers: Dict[str, str],
-        target: str, mode: str, timeseries_settings: TimeseriesSettings) -> pd.DataFrame:
+        target: str, mode: str, timeseries_settings: TimeseriesSettings,
+        anomaly_detection: bool) -> pd.DataFrame:
```

Comment:
Wrong formatting, please use the `.flake8` file's line limit to format, in this case I think the line did not go over, even if it did it's probably only the last bit